test/Frontend/load-module-with-alias.swift

```diff
@@ -0,0 +1,20 @@
+/// Test the -module-alias flag.
+
+// RUN: %empty-directory(%t)
+
+/// Create a module Bar
+// RUN: echo 'public func bar() {}' > %t/FileBar.swift
```

Comment:
Just FYI as this is a personal preference kind of thing, you may gain to use a `RUN: %{python} %utils/split_file.py -o %t %s` to break apart the test files into files ready to be compiled instead of echoing them like this.